translations/ui-data-import/en.json

```diff
@@ -23,6 +23,11 @@
   "beganRunning": "Began {time}",
   "endedRunning": "Ended {time}",
   "progressRunning": "In progress",
+  "jobProgress.partsRemaining": "Job parts remaining: {current} of {total}",
+  "jobProgress.partsProcessed": "Job parts processed: {current} of {total}",
+  "jobProgress.partsCompleted": "Completed: {amount}",
```

Comment:
Should `{amount}` be `{amount} parts`?  If I just see `Completed: 25` as a user it might be a bit confused

Reply:
for context, these are a 'subtotals' section below the parts/totals count...
![image](https://github.com/folio-org/ui-data-import/assets/20704067/26f6dbe7-d07c-4fa0-bf13-c444da28208d)

Reply:
ahh ok, sounds good!